composer.json

```diff
@@ -17,10 +17,12 @@
         {"name": "Johannes Schmitt", "email": "schmittjoh@gmail.com"}
     ],
     "require": {
-        "php": ">=5.3.2"
+        "php": "^7.2"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.5"
+        "phpunit/phpunit": "^8.2",
+        "phpstan/phpstan": "^0.11.8",
+        "doctrine/coding-standard": "^6.0"
```

Comment:
Should `config->sort-packages` be used and set to true?

Reply:
Good idea. Done.